GEOS_ChemEnvGridComp.F90

```diff
@@ -452,6 +506,47 @@ subroutine Run2 ( GC, IMPORT, EXPORT, CLOCK, RC )
        if (associated(pr_snow))  deallocate(pr_snow)
     end if
 
+!!!! >>>>>>>>>>>>>>>>  OVP
+
+    IF ( ovp_setup_done == .FALSE. ) THEN
```

Comment:
@mmanyin This needs to be:
```fortran
IF ( ovp_setup_done .eqv. .FALSE. ) THEN
```
The standard says you can't use `==` for comparing logicals.

Reply:
I say this as GNU is failing because of it (Intel doesn't seem to care 😄 )

Reply:
Thanks for the reminder!  I will update the branch.